# def display_person(*args):
#     for i in args:
#         print(i)

# display_person("Rahat", "25")




# def f1():
#     global x
#     x += 1
#     print(x)
# x = 12
# f1()
# print("x")


# def display(**kwargs):
#     for i in kwargs:
#         print(i,end=" ")

# display(emp="Kelly", salary=9000)



# try:
#     if '1' != 1:
#         raise "someError"
#     else:
#         print("someError has not occurred")
# except "someError":
#     print ("someError has occurred")


""" 


  7. When will the else part of try-except-else be executed?
a) always
b) when an exception occurs
c) when no exception occurs
d) when an exception occurs in to except block

 """


# from math import factorial
# print(math.factorial(5))
